diff --git a/node_modules/react-native-gesture-handler/ios/RNGestureHandlerManager.h b/node_modules/react-native-gesture-handler/ios/RNGestureHandlerManager.h
index a94a0fd..1e803eb 100644
--- a/node_modules/react-native-gesture-handler/ios/RNGestureHandlerManager.h
+++ b/node_modules/react-native-gesture-handler/ios/RNGestureHandlerManager.h
@@ -1,5 +1,5 @@
 #import <Foundation/Foundation.h>
-
+#import <React/RCTEventDispatcher.h>
 #import <React/RCTBridgeModule.h>
 #import <React/RCTUIManager.h>
 
